react-app/src/features/forms/post-submission/withdraw-package/index.tsx

```diff
@@ -44,12 +43,14 @@ export const WithdrawPackageActionWaiver = () => {
         documentChecker: (check) => check.recordExists,
       }}
       bannerPostSubmission={{
-        header: "Package withdrawn",
-        body: `The package ${id} has been withdrawn.`,
+        header: "Withdraw package request has been submitted",
+        body: `If CMS needs any additional information, they will follow up by email.`,
```

Comment:
Template literal is unnecessary

Reply:
Seems to be the case for lines 101, 140

Reply:
Can you put them as code suggestions. I'm lazy 😂

Reply:
I gotchu 😂